content/tutorials/reactjs-realtime-commenting.textile

```diff
@@ -22,140 +22,145 @@ tags:
     - Realtime commenting
 ---
 
-In this tutorial, we will see how to use Ably "Realtime client library":/realtime to build a realtime commenting system with React. We'll be making use Ably's public channel. Once a comment is made, we'll publish it to the public channel. We will also be subscribed to the channel so we can see comments as they are added in realtime.
+In this tutorial, we will see how to use Ably "Realtime client library":/realtime to build a realtime commenting system with React. You can see a working demo "here":http://ablycommentingdemo.infinityfreeapp.com/.
+
+We'll be making use of Ably's public channel. Once a comment is made, we'll publish it to the public channel. We will also be subscribed to the channel so we can see comments as they are added in realtime.
 
 This tutorial will not include any server-side implementation. Hence, comments won't be persisted to any storage.
 
 React is a JavaScript library for building user interfaces. It is component-based and declarative which makes it easy to build complex UI with it. For more information on React, please refer to their official website "https://reactjs.org":https://reactjs.org.
 
-<img src="/images/tutorials/react-commenting/react_logo.png" style="width: 50%" alt="React logo">
+<img src="/app/assets/images/tutorials/react-commenting/react_logo.png" style="width: 50%" alt="React logo">
 
 <%= partial partial_version('tutorials/_step-1-setup-free-account') %>
 
 h2. Step 2 - Create a React App
 
-We'll start by creating a new React app. To do this, we'll be using "create-react-app":https://github.com/facebookincubator/create-react-app. create-react-app allows you to create a React app without having to worry about build configurations. To use it, you'll need to install @create-react-app@ locally if you don't have it already:
+We’ll start by creating a new React app. To do this, we’ll be using "create-react-app":https://github.com/facebookincubator/create-react-app. The @create-react-app@ allows you to create a React app without having to worry about build configurations. To use it, you’ll need to install @create-react-app@ locally. We used version 3.4.1 and we recommend that you use the same, otherwise your experience may not match the tutorial. To install @create-react-app@  module of version 3.4.1  use:
 
-```[sh]
-npm install -g create-react-app
-```
+bc..
```

Comment:
why have you removed `sh`?

Reply:
Because "```[sh]" is a markdown syntax, while this file is .textile, so I used [textile syntax](https://textile-lang.com/doc/block-code.)

Reply:
yeah i hear you, textile is a pain, but there are two ways to do it, in textile https://github.com/ably/docs/blob/main/content/realtime/authentication.textile (basically the same as MD) or https://github.com/ably/docs/blob/main/content/client-lib-development-guide/documentation-formatting-guide.textile
Please take a moment to read the last document, it is old but still really helpful.

Reply:
Thank you for the documents. All ```[...]  are back.

Reply:
Checked, language markers are back so resolving this comment.